packages/compiler/scripts/regen-nonascii.js

```diff
@@ -14,6 +14,10 @@ const disallowedProperties = [
   "Unassigned",
 ];
 
+const disallowedCodePoints = [
```

Comment:
Should we make this a set?

Reply:
If this ever had more than a few elements, then I guess that would be worth it. But I don't see the need to do it now. This code doesn't ship. Worst case is the script gets slow when we run it at some point and we can change it then. Hopefully we'll never have a whole bunch of special cases to go in this list.

Reply:
oh yeah didn't noticed this was a script.